### PR TITLE
Added DeccanRubyConf to regional conferences

### PR DESCRIPTION
**Pull request by vipulnsward**

### CODE REVIEW

**Comment by vipulnsward** [comment]
cc @hsbt

File: en/community/conferences/index.md [hunk 1]
```diff
@@ -74,6 +74,9 @@ learn, have fun and meet fellow Rubyists.
 
 [GoRuCo][19]: New York City's annual Ruby conference. A one-day single-track conference.
 
+[DeccanRubyConf][20]: Pune(India) City's annual Ruby conference, themed around fun activities filled around the day. 
+Its a single-day single-track conference.
```
Comment (JuanitoFatas) [inline en/community/conferences/index.md]: `It is`?

File: en/community/conferences/index.md [hunk 2]
```diff
@@ -74,6 +74,9 @@ learn, have fun and meet fellow Rubyists.
 
 [GoRuCo][19]: New York City's annual Ruby conference. A one-day single-track conference.
 
+[DeccanRubyConf][20]: Pune(India) City's annual Ruby conference, themed around fun activities filled around the day.
```
Comment (JuanitoFatas) [inline en/community/conferences/index.md]: `Pune(India)` => `Pune (India)`, with a space.

**Comment by vipulnsward** [comment]
Fixed

**Comment by hsbt** [comment]
:octocat:

**Comment by vipulnsward** [comment]
@hsbt Thanks! Any idea when this would get deployed? 
Would like to link it at other places.

**Comment by JuanitoFatas** [comment]
Hi @vipulnsward,@hsbt deployed, see:

https://www.ruby-lang.org/en/community/conferences/

**Comment by vipulnsward** [comment]
Thanks!

On Friday, May 15, 2015, Juanito Fatas <notifications@github.com> wrote:

> Hi @vipulnsward <https://github.com/vipulnsward>,@hsbt
> <https://github.com/hsbt> deployed, see:
>
> https://www.ruby-lang.org/en/community/conferences/
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/ruby/www.ruby-lang.org/pull/1072#issuecomment-102246887>
> .
>


-- 
Vipul A.M.
+91-8149-204995

**Comment by stomar** [comment]
:astonished: A conference in **India**, listed on the `en` page among the regional conferences?

**Pull request merged**